test-interface/src/main/scala/org/scalajs/testinterface/internal/FrameworkDetector.scala

```diff
@@ -0,0 +1,37 @@
+package org.scalajs.testinterface.internal
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.reflect.Reflect
+
+private[internal] object FrameworkDetector {
+  @JSExportTopLevel("org.scalajs.testinterface.internal.detectFrameworks")
+  def detectFrameworks(exportsNamespace: js.Dynamic,
```

Comment:
Do we really need to inject the `exportsNamespace`? Can't we just use `scala.scalajs.runtime.environmentInfo.exportsNamespace`?